cmd/explain/explain.go

```diff
@@ -0,0 +1,70 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package explain
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+var (
+	stdout = os.Stdout
+)
+
+// Say prints an explanatory message to standard output.
+//
+// Say is only effective if Enable was called earlier.
+var Say = func(f string, args ...interface{}) {}
+
+// Header prints a spaced header, usually separating subsequent programs.
+//
+// Header  is only effective if Enable was called earlier.
+var Header = func(name string) {}
```

Comment:
given the pattern of our headers, this could take name, prefix, summary, to be rendered as:

```<< [$prefix ][$name[ ($comment)] >>```